packages/patternfly-4/content/design-guidelines/usage-and-behavior/pagination/design.md

```diff
@@ -0,0 +1,43 @@
+---
+path: "/design-guidelines/usage-and-behavior/pagination"
+---
+# Pagination
+Pagination is used when you want to break up data into multiple pages. It is typically used within a table, list or card view.  If you choose to use infinite scrolling, then pagination would not be necessary. 
+
+![top-bottom-pagination](./img/top-bottom-pagination.png)
+
+**1. Top pagination:** Pagination is always right-aligned in the toolbar. If pagination is required, we recommend using pagination both above and below content views.
+**2. Bottom pagination:** Pagination should be shown below the content view as well.
```

Comment:
You should place a line break between enumerated items otherwise this appears as one long paragraph when viewed in GitHub preview and the website.